restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -111,12 +112,12 @@
 	"dialStepSipUriInputLabel": "Sip URI",
 	"dialStepSipUriInputPlaceholder": "sip:thomas@127.0.0.1:5080",
 	"dialStepSipUriInputTooltip": "Marque para SIP URI",
-	"dialStepContinueTooltip": "Seleccione módulo que continuará al finalizar la llamada. 'core_DialCallStatus', 'core_DialCallSid', 'core_DialCallDuration' y 'core_RecordingUrl' (si es permitido grabar) las variables estarán disponibles.",
+	"dialStepContinueTooltip": "Seleccione módulo que continuará al finalizar la llamada. Las siguientes variables estarán disponibles (si esta permitido grabar): 'core_DialCallStatus', 'core_DialCallSid', 'core_DialCallDuration' y 'core_RecordingUrl'.",
 	"dialStepContinueLabel": "Continuar con",
 	"dialStepContinueTimeoutTooltip": "Límite de segundos que la persona que llama espera para que contesten",
 	"dialStepContinueTimeoutLabel": "Se agotó el tiempo",
 	"dialStepContinueTimeoutInputPlaceholder": "30",
-	"dialStepContinueTimeLimitTooltip": "Máximo duración de segundos para la persona que llama",
+	"dialStepContinueTimeLimitTooltip": "Máximo duración de segundos para marcar",

```

Comment:
Grammar error found. Change for:
"dialStepContinueTimeLimitTooltip": "Máxima duración de segundos para marcar",